proofs/logic.ml

```diff
@@ -397,14 +398,15 @@ let rec mk_refgoals ~check env sigma goalacc conclty trm =
         let ty = EConstr.Unsafe.to_constr ty in
           (acc',ty,sigma,c)
 
-      | Case (ci,p,c,lf) ->
+      | Case (ci,p,iv,c,lf) ->
+        (* XXX Is ignoring iv OK? *)
```

Comment:
I have no clue what this function and the one a few lines below do, so I don't know if the case_invert data should be ignored or something should be done with it.